src/main/kotlin/org/jitsi/jibri/api/http/HttpApi.kt

```diff
@@ -90,7 +90,7 @@ class HttpApi(private val jibriManager: JibriManager) {
                 val callLoginParams = startServiceParams.callLoginParams ?: return@run StartServiceResult.ERROR
                 jibriManager.startFileRecording(
                     ServiceParams(usageTimeoutMinutes = 0),
-                    FileRecordingRequestParams(startServiceParams.callParams, callLoginParams),
+                    FileRecordingRequestParams(startServiceParams.callParams, "TODO: sessionID", callLoginParams),
```

Comment:
the HTTP API is not used in production right now, so these TODOs are ok.  I have a TODO list item to come back and get this to parity with the XmppApi, but really right now it's just useful for easier testing of Jibri.